app/src/main/res/layout/fragment_currency_details.xml

```diff
@@ -0,0 +1,194 @@
+            tools:text="BTC/MXN" />
+
+        <TextView
+            android:id="@+id/detail_currency_trades"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/margin_items"
+            android:text="@string/latest_market_trades"
+            android:textColor="@color/darkGray"
+            android:textSize="@dimen/subtitle_size"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/detail_currency_market" />
+
+        <TextView
+            android:id="@+id/detail_currency_tab_bids"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/margin_items"
+            android:textSize="@dimen/text_subtitle"
+            android:textStyle="bold"
+            android:textColor="@color/darkGray"
+            android:paddingEnd="@dimen/layout_padding"
+            android:paddingStart="@dimen/layout_padding"
+            android:background="@drawable/borderbottom"
+            android:clickable="true"
+            android:focusable="true"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintEnd_toStartOf="@+id/detail_currency_tab_asks"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/detail_currency_trades"
+            android:text="@string/bids" />
+
+        <TextView
+            android:id="@+id/detail_currency_tab_asks"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:paddingEnd="@dimen/layout_padding"
+            android:paddingStart="@dimen/layout_padding"
+            android:textColor="@color/darkGray"
+            android:textSize="@dimen/text_subtitle"
+            android:textStyle="bold"
+            android:clickable="true"
+            android:focusable="true"
+            app:layout_constraintBaseline_toBaselineOf="@+id/detail_currency_tab_bids"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toEndOf="@+id/detail_currency_tab_bids"
+            android:text="@string/asks" />
+
+        <androidx.recyclerview.widget.RecyclerView
+            android:id="@+id/trades_list"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="@dimen/margin_items_cards"
+            android:clipToPadding="false"
+            android:orientation="vertical"
+            app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"
+            app:layout_constraintTop_toBottomOf="@id/detail_currency_tab_bids"
+            tools:itemCount="10"
+            tools:layout_editor_absoluteX="16dp"
+            tools:listitem="@layout/item_trades" />
+
+        <androidx.constraintlayout.widget.Guideline
```

Comment:
I was wondering what is the purpose of this guideline :)

Reply:
Forgot to delete it while I was doing tests with constraint layout 😸